tests/component_integration/conftest.py

```
@@ -26,10 +26,12 @@
 from unittest.mock import patch
 
 # Limit glibc malloc arenas to avoid heap-corruption SIGABRT and OOM-killed
-# xdist workers under heavy `-n auto` load. Each component_integration test
-# runs aiperf in-process with full Pydantic / msgspec / tokenizer imports,
-# so the default 8×NCPU arenas per worker × 24 workers blows out RAM. The
-# integration conftest carries the same setting (gotcha 2026-04-21).
+# xdist workers under heavy `-n auto` load. Component_integration runs aiperf
+# in-process with full Pydantic / msgspec / tokenizer / torch imports, so the
+# default 8×NCPU arenas blows out RAM in 2-CPU CI runners. glibc reads this
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)